pynumdiff/utils/evaluate.py

```diff
@@ -48,8 +48,8 @@ def plot(x, dt, x_hat, dxdt_hat, x_truth, dxdt_truth, xlim=None, show_error=True
     axes[1].legend(loc='lower right', fontsize=12)
 
     if show_error:
-        _, _, rms_dxdt = rmse(x, dt, x_hat, dxdt_hat, x_truth, dxdt_truth)
-        R_sqr = error_correlation(dxdt_hat, dxdt_truth)
+        rms_dxdt = rmse(dxdt_truth, dxdt_hat)
+        R_sqr = error_correlation(dxdt_truth, dxdt_hat)
```

Comment:
Changed the order of inputs to all functions to have known stuff first and hatted, computed stuff second.